File: matlab/ine/ine_jaxi_example.m
% Ejemplo mínimo de uso para descargar una tabla de INE JAXI y exportar a CSV

% Cambiar el directorio de trabajo a la carpeta de este archivo
cd(pwd);

% Parámetros de ejemplo (tabla 67821, sin cabeceras adicionales)
tableId = '67821';
nocab = '1';

% Llamar a la función que devuelve una tabla de MATLAB
T = ine_jaxi_function(tableId, nocab);

% Construir ruta de salida
outPath = 'ine_jaxi_example.csv';

% Exportar como CSV con punto y coma como delimitador y UTF-8
writetable(T, outPath, 'Delimiter', ';', 'FileType', 'text');

File: R/ine/ine_jaxi_example.R
# Ejemplo de uso de la función ine_jaxi_api_function
library(httr)

#setwd("ine")
source("ine_jaxi_function.R")

# Ejemplo de uso
df <- ine_jaxi_api_function(
  tableId = "67821",
  nocab = "1"
)

write.csv(df, "ine_jaxi_example.csv", row.names = FALSE)


